package.json

```diff
@@ -2,7 +2,7 @@
   "name": "@hcl-software/domino-rest-adminclient",
   "version": "1.30.7-SNAPSHOT",
   "private": false,
-  "proxy": "http://localhost:8880",
+  "proxy": "https://frascati.projectkeep.io",
```

Comment:
Frascati should not be referred to, it has nothing to do with our offering.  not positive what proxy does for package.json, but whatever it is we shouldn't refer to frascati.